embedded-workshop-book/src/listing-usb-devices.md

```diff
@@ -1,23 +1,25 @@
 # Listing USB Devices
 
-In the `tools` folder you'll find `usb-list`: a minimal cross-platform version of the `lsusb` tool. Run it (`cargo run` from `tools/usb-list`) to list all USB devices.
+In the `tools` folder you'll find `usb-list`: a minimal cross-platform version of the `lsusb` tool. You have already installed it; run it to list all USB devices.
```

Comment:
if they've already installed it I don't think we need to point them to the tools folder. I'd rather say something along the lines of "run usb-list– if you haven't installed it yet look at [this section](#section) on how to catch up"